src/main/java/com/czertainly/api/model/core/oid/SystemOid.java

```diff
@@ -0,0 +1,88 @@
+package com.czertainly.api.model.core.oid;
+
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Getter
+public enum SystemOid {
+
+    // RDN Attribute Type
+    EMAIL("1.2.840.113549.1.9.1", "Email", OidCategory.RDN_ATTRIBUTE_TYPE, "EMAIL", List.of("E", "EMAILADDRESS")),
+    COMMON_NAME("2.5.4.3", "Common Name", OidCategory.RDN_ATTRIBUTE_TYPE, "CN", List.of()),
+    ORGANIZATION_UNIT("2.5.4.11", "Organization Unit", OidCategory.RDN_ATTRIBUTE_TYPE, "OU", List.of()),
+    ORGANIZATION("2.5.4.10", "Organization", OidCategory.RDN_ATTRIBUTE_TYPE, "O", List.of()),
+    LOCALITY("2.5.4.7", "Locality", OidCategory.RDN_ATTRIBUTE_TYPE, "L", List.of()),
+    STATE("2.5.4.8", "State", OidCategory.RDN_ATTRIBUTE_TYPE, "ST", List.of("S")),
+    DOMAIN_COMPONENT("0.9.2342.19200300.100.1.25", "Domain Component", OidCategory.RDN_ATTRIBUTE_TYPE, "DC", List.of()),
+    COUNTRY("2.5.4.6", "Country", OidCategory.RDN_ATTRIBUTE_TYPE, "C", List.of()),
+    DISTINGUISHED_NAME_QUALIFIER("2.5.4.46", "Distinguished Name Qualifier", OidCategory.RDN_ATTRIBUTE_TYPE, "DNQ", List.of()),
+    TITLE("2.5.4.12", "Title", OidCategory.RDN_ATTRIBUTE_TYPE, "T", List.of("TITLE")),
+    SURNAME("2.5.4.4", "Surname", OidCategory.RDN_ATTRIBUTE_TYPE, "SN", List.of("SURNAME")),
+    GIVEN_NAME("2.5.4.42", "Given Name", OidCategory.RDN_ATTRIBUTE_TYPE, "GIVENNAME", List.of()),
+    INITIALS("2.5.4.43", "Initials", OidCategory.RDN_ATTRIBUTE_TYPE, "INITIALS", List.of()),
+    PSEUDONYM("2.5.4.65", "Pseudonym", OidCategory.RDN_ATTRIBUTE_TYPE, "PSEUDONYM", List.of()),
+    GENERATION_QUALIFIER("2.5.4.44", "Generation Qualifier", OidCategory.RDN_ATTRIBUTE_TYPE, "GENERATION", List.of()),
+
+    // Extended Key Usage
```

Comment:
What about other standardized EKUs? Are they going to be displayed properly when not included in this list? The same for other OIDs, RDNs, SANs, etc.